api.raml

```diff
@@ -352,6 +361,22 @@ schemas:
           description: |
             The corresponding user record (matching on the `employee_id` URL
             parameter) was updated to match the given data.
+          body:
+            application/json:
+              schema: User
+              example: |
+                {
+                  "employee_id": "12345",
+                  "first_name": "John",
+                  "last_name": "Smith",
+                  "display_name": "John Smith",
+                  "username": "john_smith",
+                  "email": "john_smith@example.com",
+                  "active": "yes",
```

Comment:
yep, that's better:  https://github.com/silinternational/idp-id-broker/commit/4eb6316744ab7da8da0c32f06af6ec23b3a554b1